artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/JsonUtil.java

```diff
@@ -111,14 +108,13 @@ public static Object[] fromJsonArray(final JsonArray jsonArray) throws Exception
             }
 
             array[i] = map;
-         } else if (val instanceof JsonString) {
-            array[i] = ((JsonString) val).getString();
+         } else if (val instanceof JsonString string) {
+            array[i] = string.getString();
```

Comment:
```suggestion
         } else if (val instanceof JsonString jsonString) {
            array[i] = jsonString.getString();
```

Reply:
I'm giving up looking at the rest of the json stuff...same type of recent comments+suggestions to make it more readable than originally, rather than arguably less readable, also apply to the rest of it as well.

Reply:
I think I got all of them.